packages/cli-hydrogen/src/cli/services/deploy.ts

```diff
@@ -52,66 +50,51 @@ export async function deployToOxygen(_config: DeployConfig) {
           assetBaseURL: ctx.assetBaseURL,
         })
 
-        return task.newListr(subTasks)
+        return subTasks
       },
-      skip: (ctx) => ctx.config.pathToBuild,
+      skip: (ctx) => Boolean(ctx.config.pathToBuild),
     },
     {
-      title: '🚀 Uploading deployment files',
+      title: 'Uploading deployment files',
       task: async (ctx, task) => {
         await shouldRetryOxygenCall(task, 'Uploading files to Oxygen failed.')
 
         ctx.previewURL = await uploadDeployment(ctx.config, ctx.deploymentID)
-        task.output = `Preview URL: ${ctx.previewURL}`
-        task.title = '🚀 Files uploaded'
-      },
-      options: {
-        bottomBar: Infinity,
-        persistentOutput: true,
       },
       retry: backoffPolicy.length,
     },
     {
-      title: '📡 Checking deployment health',
+      title: 'Checking deployment health',
       task: async (ctx, task) => {
-        const retryCount = task.isRetrying()?.count
+        const retryCount = task.retryCount
 
         if (retryCount === backoffPolicy.length) {
-          task.title =
-            "The deployment uploaded but hasn't become reachable within 2 minutes. Check the preview URL to see if deployment succeeded. If it didn't, then try again later."
-          return
+          throw new Error(
```

Comment:
Not sure if this should be an error. Maybe we should just return and show a warning message at the end. We can set something in the `ctx` and conditionally show the warning banner.